src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -229,18 +235,26 @@ bool ClusterTaskManager::CancelTask(
     }
   }
 
-  return local_task_manager_->CancelTask(
-      task_id, failure_type, scheduling_failure_message);
+  if (local_task_manager_) {
+    return local_task_manager_->CancelTask(
+        task_id, failure_type, scheduling_failure_message);
+  } else {
+    return false;
+  }
 }
 
 void ClusterTaskManager::FillPendingActorInfo(rpc::GetNodeStatsReply *reply) const {
-  scheduler_resource_reporter_.FillPendingActorInfo(reply);
+  if (scheduler_resource_reporter_) {
+    scheduler_resource_reporter_->FillPendingActorInfo(reply);
+  }
 }
 
 void ClusterTaskManager::FillResourceUsage(
```

Comment:
Where is the code where we report infeasible actors from gcs cluster_task_manager to the autoscaler? Autoscaler needs to know the resource demand so it can launch new nodes.

Reply:
I'm going to handle this in the next PR, when we enabling gcs scheduling to passing the entire CI pipeline.